### ISSUE TITLE
TF2 - Impossible to track robot kills from sentry guns. (Broken for cosmetics)

### ISSUE THREAD

**Issue by jh34ghu43gu** [ValveSoftware/Source-1-Games#3558]
There is currently no way for robot kills via an unwrangled sentry gun to count for any strange item with qualified parts.

- Wrangler parts only track when wrangled
- Wrench parts only track wrench melee kills
- Cosmetic parts only track wrangled kills*
- Strange PDA cannot have parts applied to it

*I believe this one is a bug and the other two (not the wrangler one) would be feature requests, as strange engineer cosmetics still count points from sentry kills (unsure about the cosmetic kills part, no item to test with).

**Comment by ConnorBrennan** [comment]
You should be able to put the "robots killed" strange part on the PDA

**Comment by jh34ghu43gu** [comment]
> You should be able to put the "robots killed" strange part on the PDA

You cannot put any strange parts on a strange PDA. This may also be a bug or intentional from the 3 applied parts (sentry kills, teleports, dispensed health) counting to the part cap.

**Comment by FrozenDragon0** [comment]
> > You should be able to put the "robots killed" strange part on the PDA
> 
> You cannot put any strange parts on a strange PDA. This may also be a bug or intentional from the 3 applied parts (sentry kills, teleports, dispensed health) counting to the part cap.

Pretty sure it's a bug, as some other strange items with pre-applied parts such as the Strange Loose Cannon with it's double donk part doesn't count towards the part cap and still allows for 3 strange parts to be applied.

**Comment by LordScout** [comment]
no way dude 2 years and nobody has fixed it??

**Comment by M15CH1EF** [comment]
I applied a bunch of strange parts to my wrench expecting it to track what my sentry kills... Now I understand why some wrenches with said parts sell for really low prices, lol.

**Comment by Petster** [comment]
Yes this needs to be fixed as soon as possible for the new MvM update. Us MvM engineers need to track robots destroyed via sentry. Thank you
